### PR TITLE
[MIG] migrate business_requirement_deliverable_resource_template_categ from 10.0 to 12.0

### PR DESCRIPTION
**Pull request by ruter-lyu**

### CODE REVIEW

File: business_requirement_deliverable_resource_template_categ/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# © 2016-2019 Elico Corp (https://www.elico-corp.com).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Business Requirement Deliverable Resource Template Category',
+    'category': 'Business Requirements Management',
+    'summary': """Manage default resource lines categories in your
+               deliverable templates""",
+    'version': '12.0.1.1.0',
```
Comment (elicoidal) [inline business_requirement_deliverable_resource_template_categ/__manifest__.py:8]:
```suggestion
    'version': '12.0.1.0.0',
```

**Review by elicoidal — COMMENTED** [review]
almost there. Please prepare the README

**Comment by elicoidal** [comment]
@JoJoJoJoJoJoJo @joycehuan @seb-elico please have a look

**Review by JoJoJoJoJoJoJo — APPROVED** [review]
LGTM, follow review of @elicoidal

File: business_requirement_deliverable_resource_template_categ/README.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
+.. figure:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (nikul-serpentcs) [inline business_requirement_deliverable_resource_template_categ/README.rst:3]: @ruter-lyu LGPL-3.0

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
netpicking

**Comment by pedrobaeza** [comment]
Closing as old and not green

**Pull request closed without merging**